docs/examples/testing/unit_test.py

```diff
@@ -6,6 +6,8 @@
 
 
 class ConversionService(object):
+    """ Service under test
+    """
     math = RpcProxy("math_service")
```

Comment:
Also, if we encourage the convention, we might want to do a pass on these examples adding the `_rpc` suffix to proxy declarations